content/frame/benchmarking-1/slides.md

```diff
@@ -267,11 +267,12 @@ For each component and repeat:
 - We set number of steps to 3.
 - Vary one component at a time, select high value for the others.
 
-|     | Δx  | Δy  | Δy  | Δz  | Δz  | max |
-| --- | --- | --- | --- | --- | --- | --- |
-| x   | 1   | 2   | 2   | 2   | 2   | 2   |
-| y   | 5   | 0   | 2   | 5   | 5   | 5   |
-| z   | 10  | 10  | 10  | 0   | 5   | 10  |
+<!-- prettier-ignore -->
+|   | Δx | Δy | Δy | Δz | Δz | max |
+| - | -- | -- | -- | -- | -- | --- |
+| x | 1  | 2  | 2  | 2  | 2  | 2   |
+| y | 5  | 0  | 2  | 5  | 5  | 5   |
+| z | 10 | 10 | 10 | 0  | 5  | 10  |
```

Comment:
prettier and dprint conflict on a few things, we want to run this on tables in almost all cases, but when these flop back and forth, use dprint and then ignore the item.